docs/components/segmentedBar.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebarDepth: 0
+---
```

Comment:
there should be a live demo section where users can adjust all props

Reply:
I've added a demo section